src/main/java/org/fundacion/jala/converter/models/AssetSQL.java

```diff
@@ -62,9 +66,9 @@ public static void editProjectData(final int projectId, final String projectName
     }
 
     /**
-     * Deletes a project.
+     * Deletes a date from the database of the audio or video converter.
```

Comment:
It would be better to put:
Deletes data of the audio or video converter from the database.

Reply:
thanks for the suggestion

Reply:
it was solved